metaflow/metaflow_config.py

```diff
@@ -46,11 +46,14 @@ def from_conf(name, default=None):
 DEFAULT_METADATA = from_conf('METAFLOW_DEFAULT_METADATA', 'local')
 METAFLOW_USER = from_conf('METAFLOW_USER')
 
+METAFLOW_USER = from_conf('METAFLOW_USER')
```

Comment:
Apologies, there seem to be some spacing issues because of `black` formatting. I'll fix them once everyone has submitted their approval to avoid sending lots of notifications.